.travis.yml

```diff
@@ -1,21 +0,0 @@
-language: node_js
-node_js:
-  - "10"
-  - "12"
-  - "14"
-
-env:
-  - CXX=g++-4.8
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-      - mongodb-3.2-precise
-    packages:
-      - g++-4.8
-      - mongodb-org-server
-      - mongodb-org-shell
-services:
-  - mongodb
-
-after_success: npm run coverage
```

Comment:
I believe this command is publishing code coverage data to https://coveralls.io/github/strongloop/loopback-connector-mongodb. Can you please preserve this step in the new GHA workflow too?

I think it should be enough to add a new step at the end of the file:

```
    - run: npm install
    - run: npm test
      env:
        CI: true
    - run: npm run coverage
```